fenix/app/src/main/java/org/mozilla/fenix/browser/tabstrip/TabStrip.kt

```diff
@@ -279,22 +284,37 @@ private fun TabItem(
             horizontalArrangement = Arrangement.SpaceBetween,
         ) {
             Row(
-                modifier = Modifier.weight(1f, fill = false),
+                modifier = Modifier.weight(1f),
                 verticalAlignment = Alignment.CenterVertically,
             ) {
+                // This makes sure that isRtl is only calculated when the title changes.
```

Comment:
this comment may not be super necessary after this review

Reply:
Yeah added after to make it more understandable, guessing it was useful for the review so maybe it's useful for future readers 😄  seeing we don't have a lot of remember usages for this